src/app/gestao-usuarios/cadastro/individual/page.jsx

```
@@ -1,263 +1,287 @@
+		if (typeof perfilAtivo === "boolean" && perfilAtivo === false) {
+			return "Não";
+		}
+
+		return "Primeiro acesso pendente";
+	};
+
+	useEffect(() => {
+		const transformarDadosEmLinhas = (dados) => {
+			return dados.map((dado) => ({
+				id: uuidV4(),
+				usuarioId: dado["id_usuario"],
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)